weis/inputs/analysis_schema.yaml

```diff
@@ -499,7 +499,47 @@ properties:
                         description: Constrain design to one where OpenFAST simulations don't fail_value
                         default: False
 
-    
+    driver:
+        type: object
+        default: {}
+        properties:
+            design_of_experiments:
+                type: object
+                description: Specification of the design of experiments driver parameters
+                default: {}
+                properties:
+                    skip_if_results_exist:
+                        type: boolean
+                        default: True
+                        description: Toggle skipping the design of experiments if results exist
+                    train_surrogate_model:
+                        type: boolean
+                        default: False
+                        description: Train surrogate model after running design of experiment
+            surrogate_based_optimization:
```

Comment:
May move this outside of WEIS/runWEIS.

Surrogate optimization is a different workflow, that will be run multiple times, whereas runWEIS is a one-time event.

Since it's not part of this PR, we'll probably remove.

Reply:
Removed surrogate_based_optimization from the analysis_schema